### PR TITLE
feat: Update CardLayout component [BLOX-166]

### PR DESCRIPTION
**Pull request by Kristina2103**
# Pull Request Template

Feel free to remove any section (except the checklist) from this template.
But please try to fill in as much of this as possible.

## Description

Figma https://www.figma.com/file/8G6OqCH6vmpyreKf60KIcV/BloxiFi---public?node-id=47%3A130
## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a configuration update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide easily reproducible instructions.
List any relevant details for your test configuration.

## Checklist:

- [ ] I have performed a self-review of this code and spelling
- [ ] I have added comments in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Additional information

Include any additional information about these changes here.

For example, if you are making a visual change, please include BEFORE (if any) and AFTER screenshots.
A visual change could be just reordering some components on a page.
Any visual changes in the UI MUST provide these screenshots.

### CODE REVIEW

File: packages/ui/src/styles/colors.ts [hunk 1]
```diff
@@ -2,7 +2,11 @@ export const Colors = {
   headerBackground: '#F6F8FA',
   logoBackground: '#57606A',
   background: '#FFFFFF',
+
+  //Borders
   borderColor: '#D0D7DE',
+  borderLight: 'rgba(0,0,0,0.2)',
```
Comment (aleksaToljic) [inline packages/ui/src/styles/colors.ts:3]: we need to add every color in light and dark objects. 
If I missed some new color that you added, please add it like this.

```tsx
light: {someColor: 'color'}
dark: {someColor: 'color'}
```

They should have the same naming (and initially the same color, but later on we will change  to `dark: {someColor: 'darkColor'}`

**Review by aleksaToljic — CHANGES_REQUESTED** [review]

File: packages/ui/src/styles/colors.ts (hunk 1 above)
Comment (Kristina2103) [reply packages/ui/src/styles/colors.ts:3]: @aleksaToljic should I also change imports in components for those colors, to get the color of active theme mode  or just update light/dark objects for now?

Comment (aleksaToljic) [reply packages/ui/src/styles/colors.ts:3]: well, on homePage, I used other approuch i fetched what theme we are currently using
https://styled-components.com/docs/advanced

Comment (aleksaToljic) [reply packages/ui/src/styles/colors.ts:3]: although we should probably use a bit better approuch for this, but we can work with this for now, if you think of something better let me know.

**Review by aleksaToljic — APPROVED** [review]

**Pull request merged**